src/static/simple.css

```diff
@@ -721,4 +721,56 @@ sub {
   margin: 2rem 0;
   overflow-x: auto;
   padding: 1.5rem;
-}
\ No newline at end of file
+}
```

Comment:
This is a UI thing (and therefore not really my area of expertise) but I prefer the way the form looks when the styling from line 724 onward is not there. The point of [simple.css](https://simplecss.org/) is to not have to use classes (it's a "classless" stylesheet). Also I suspect Matt and Courtney might have UI suggestions/tweaks later on, so my preference would be to keep the HTML and the CSS as simple as possible and then tweak it later as needed.

Reply:
Also, on mobile (not that we need to worry too much about mobile) it looks a bit scrunched up:

![Screenshot 2024-07-02 at 12 26 21 PM](https://github.com/ClinGen/stanford-affils/assets/60749003/954d52be-9d52-45d5-9a80-b4d167751500)

Versus:

![Screenshot 2024-07-02 at 12 28 47 PM](https://github.com/ClinGen/stanford-affils/assets/60749003/b8e607de-a307-4d2e-9c4d-7003fb5f5834)

Reply:
In the second screenshot we would need to add some space between the the submit and edit buttons and the blue header.

Reply:
Happy to remove the styling! I think differentiating color between buttons (Submit vs Cancel) and eventually Add/Edit/Delete/View could be beneficial to a user for ease of use. However, that is a personal preference of mine and I am happy to keep it with the base styling!

Reply:
Added the requested spacing (see screenshot below):

![Screenshot 2024-07-02 at 1 40 40 PM](https://github.com/ClinGen/stanford-affils/assets/169506092/96103e87-5801-4cf8-b7be-e65e867a7c4e)